com.github.rajsolai.textsnatcher.yml

```diff
@@ -0,0 +1,27 @@
+app-id: com.github.rajsolai.textsnatcher
+
+runtime: org.gnome.Platform
+runtime-version: "40"
+base: io.elementary.BaseApp
+base-version: "juno-20.08"
```

Comment:
```suggestion
base-version: "juno-21.08"
```

Reply:
I didn't know of a newer base app version thanks , I will update :+1: